app/views/subscriptions/index.html.erb

```diff
@@ -31,6 +31,10 @@
           user_subscriptions_path(@user, type: "works"),
           @subscribable_type == "works") %>
   </li>
+  <li>
+  <%= link_to t(".navigation.delete_all"),
```

Comment:
For clarity, can you make the links on the various subscribable type pages include the type in their text, e.g. "Delete All Work Subscriptions"?